challenges/xml/2019_US.xml

```diff
@@ -637,271 +625,255 @@
             <option name="2" description="2" />
             <option name="3" description="3" />
         </objective-enum>
-        <objective-enum id="M12-2" description="M12-scoring2" default="0" resource="building-unit">
-            <option name="0" description="0" resource-use="0" />
-            <option name="1" description="1" resource-use="10" />
-            <option name="2" description="2" resource-use="20" />
-        </objective-enum>
-        <objective-enum id="M12-3" description="M12-scoring3" default="0" resource="building-unit">
-            <option name="0" description="0" resource-use="0" />
-            <option name="1" description="1" resource-use="1" />
-            <option name="2" description="2" resource-use="2" />
-            <option name="3" description="3" resource-use="3" />
-            <option name="4" description="4" resource-use="4" />
-            <option name="5" description="5" resource-use="5" />
-            <option name="6" description="6" resource-use="6" />
-            <option name="7" description="7" resource-use="7" />
-            <option name="8" description="8" resource-use="8" />
-            <option name="9" description="9" resource-use="9" />
-        </objective-enum>
+        <objective-number id="M12-4" description="M12-scoring4" min="0" max="29" default="0" resource="building-unit" resource-use="0.5" />
```

Comment:
Why 0.5?

Reply:
Mission 12 asks the teams to build stacks out of building units.  The question asks what the total height of the stacks is.  Depending on the geometry of the stack, adding a unit will add either 2, 1, or 0 to the height of the stack.  Since 12 of the units can add 2 levels of height to the stack total, the closest approximation to the number of building units used is height/2.  That's the minimum possible number of units being used, so it's the amount to claim against the resource.